### PR TITLE
feat(module.js): export initialized class, clean up

### PR DESCRIPTION
**Pull request by piotrmichal**

### CODE REVIEW

File: src/core/init.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Linker from './linker';
 import registry from './registry';
 import $ from '../dom/element';
 import { attachNewInitObserver, attachNewTeardownObserver } from './observer';
-import config from '../config';
+import { initializedSelector } from '../module';
```
Comment (mateuszluczak) [inline src/core/init.js]: I think referencing to `module` makes the imports diagram messy, maybe it's better to move `initializedSelctor` to config. `module` should never be used as internal dependency

File: src/core/linker.js [hunk 2]
```diff
@@ -1,9 +1,7 @@
 import $ from '../dom/element';
-import config from '../config';
+import { initializedSelector } from '../module';
```
Comment (mateuszluczak) [inline src/core/linker.js]: Same here

**Comment by mateuszluczak** [comment]
@piotrmichal after discussion with @xolir there is are few things that would be nice to do in this ticket:

1. Considering the current usage:
```
import { initalisedSelector, Component } from 'strudel';
```

It may seem like a function to be used, not really a constant.

Please modify the constants that are exported by the library - version, class and selector to follow the constant naming convention, for example
`initialisedSelector` ➡️ `INIT_SELECTOR`
`initialisedClassName` ➡️ `INIT_CLASS`
`version` ➡️`VERSION`

```
import { INIT_SELECTOR, Component } from 'strudel';
```
is much more clear

2. Please provide unit test to check if these constants are properly exported

3. Please provide documentation for these exported constants in https://github.com/strudeljs/strudeljs.org

**Comment by xolir** [comment]
Closing in favour of https://github.com/strudeljs/strudel/pull/97

**Pull request closed without merging**